source/types/options.ts

```diff
@@ -251,8 +252,8 @@ Normalized options passed to the `fetch` call and the `beforeRequest` hooks.
 */
 export interface NormalizedOptions extends RequestInit { // eslint-disable-line @typescript-eslint/consistent-type-definitions -- This must stay an interface so that it can be extended outside of Ky for use in `ky.create`.
 	// Extended from `RequestInit`, but ensured to be set (not optional).
-	method: RequestInit['method'];
-	credentials: RequestInit['credentials'];
+	method: NonNullable<RequestInit['method']>;
+	credentials: NonNullable<RequestInit['credentials']>;
```

Comment:
`exactOptionalPropertyTypes: true` fails on this, as optional properties get resolved as `Foo | undefined`, and that isn't compatible with `RequestInit`.